packages/x-components/src/shims-tsx.d.ts

```diff
@@ -4,8 +4,5 @@ declare global {
   namespace JSX {
     interface Element extends VNode {}
     interface ElementClass extends Vue {}
-    interface IntrinsicElements {
-      [elem: string]: any;
-    }
```

Comment:
We don't use JSX but this is already  a record type in the base JSX definition

Reply:
I think we can totally get rid of this file. Improting from Vue will now load all the jsx custom types, which is the issue why it conflicts with React